lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -381,6 +381,21 @@ private VerifiableCredentialBundle getVerifiableCredentials(
                                 evcsIdentityVcs,
                                 isPendingEvcs,
                                 vcBundle.isF2fIdentity());
+
+                if (hasPartiallyMigratedVcs) {
```

Comment:
I was thinking we could just do:
```
if (hasPartiallyMigratedVcs) {
  return new VerifiableCredentialBundle(tacticalVcs, true, true);
}
```
i.e. force it to use the tactical pending VCs if the EVCS set is incomplete.

I think that's effectively what you're trying to do, but keeps the 'partial' logic separate, and hopefully clearer too?

Reply:
Yeh, I think I got a bit confused :) now updated